src/ClientSideMonitoring/ConfigurationProvider.php

```diff
@@ -182,7 +182,7 @@ public static function ini($profile = null, $filename = null)
             }
 
             // port is optional
-            if (empty($data[$profile]['csm_port'])) {
+            if (empty($data[$profile]['csm_port']) || filter_var($data[$profile]['csm_port'], FILTER_VALIDATE_INT) === false) {
```

Comment:
This should be multi-line:

```suggestion
            if (empty($data[$profile]['csm_port']) 
                || filter_var($data[$profile]['csm_port'], FILTER_VALIDATE_INT) === false
            ) {
```